pkg/constants/constants.go

```diff
@@ -23,8 +23,8 @@ import (
 
 const (
 	BiosPartName       = "bios"
-	EfiLabel           = "COS_GRUB"
-	EfiPartName        = "efi"
+	BootLabel          = "COS_GRUB"
+	BootPartName       = "efi"
```

Comment:
Note this PR changes the variable and constant names, but none of the values. This to ensure no regressions are introduced. If we consider changes values to I'd do it in a separate and follow up PR.